packages/chat-component/src/style.ts

```diff
@@ -403,6 +403,13 @@ export const mainStyle = css`
     margin-top: 5px;
     font-size: small;
   }
+  .items__listItem--citation.active {
+    background-color: var(--accent-high);
+
+    .items__link {
```

Comment:
nesting is only supported in pure CSS if there's `&` before nested classes

Reply:
fixed